lib/bolt/cli.rb

```diff
@@ -390,7 +391,14 @@ def execute(options)
       end
 
       if options[:mode] == 'task' && options[:action] == 'show'
-        outputter.print_table(list_tasks)
+        if options[:object]
+          outputter.print_task(get_task_info(options[:object]))
+        else
+          outputter.print_table(list_tasks)
+          outputter.print_message("Use `bolt task show <task-name>` to view
```

Comment:
this `bolt task show <task>` elsewhere it should probably consistent

Reply:
Not sure what you mean? Where else should I put the string?

Reply:
Updated the comment for clarity.